ownCloudAppShared/Client/Collection Views/Cells/DriveHeaderCell.swift

```diff
@@ -38,6 +38,34 @@ class DriveHeaderCell: DriveListCell {
 		coverObservation?.invalidate()
 	}
 
+	override init(frame: CGRect) {
+		super.init(frame: frame)
+		setupHighContrastMode()
+		updateForContrastMode()
+	}
+	
+	@MainActor required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	var contrastObservation: NSObjectProtocol?
+
+	func setupHighContrastMode() {
+		contrastObservation = NotificationCenter.default.addObserver(forName: UIAccessibility.darkerSystemColorsStatusDidChangeNotification, object: nil, queue: .main, using: { [weak self] _ in
```

Comment:
remove observer is missing on deinit

Reply:
This is no longer necessary as observers added via this method are automatically removed once the returned `observation` object is deallocated. As the returned `contrastObservation` object is released and deallocated at the time `DriveHeaderCell` is deinited/deallocated, the observation is automatically removed.